### PR TITLE
Fixed bad redirect from #9572

### PR DESCRIPTION
**Pull request by rpetrusha**
## Fixed bad redirect from #9572

Related to #9572

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -643,7 +643,7 @@
             "redirect_url": "https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki"
         },
         {
-            "source_path": "docs/framework/standard/design-guidelines/dispose-pattern.md",
+            "source_path": "docs/standard/design-guidelines/dispose-pattern.md",
             "redirect_url": "~/dotnet/standard/garbage-collection/implementing-dispose"
```
Comment (pkulikov) [inline .openpublishing.redirection.json]: @rpetrusha I've never seen the `redirect_url` starting with `~`. Is it equivalent to
```json
"redirect_url": "/dotnet/standard/garbage-collection/implementing-dispose"
```
?

Comment (mairaw) [inline .openpublishing.redirection.json]:
```suggestion
            "redirect_url": "/dotnet/standard/garbage-collection/implementing-dispose"
```

Comment (mairaw) [reply .openpublishing.redirection.json]: You're right @pkulikov. It shouldn't.

**Review by mairaw — APPROVED** [review]
LGTM. Just fix the redirect url.

**Comment by rpetrusha** [comment]
Thanks for pointing that out, @pkulikov. I've made the fix, @mairaw..

**Pull request merged**